admin_guide/cluster_capacity.adoc

```diff
@@ -0,0 +1,160 @@
+that is used by the tool for estimation. The `podspec` specifies its resource requirements as
+`limits` or `requests`. The cluster capacity tool takes the pod's resource requirements into account
+for its estimation analysis. An example pod spec is give below:
+
+```
+apiVersion: v1
+kind: Pod
+metadata:
+  name: small-pod
+  labels:
+    app: guestbook
+    tier: frontend
+spec:
+  containers:
+  - name: php-redis
+    image: gcr.io/google-samples/gb-frontend:v4
+    imagePullPolicy: Always
+    resources:
+      limits:
+        cpu: 150m
+        memory: 100Mi
+      requests:
+        cpu: 150m
+        memory: 100Mi
+```
+
+
+The output of the cluster capacity command is a number of estimated pods that can be scheduled on the cluster.
+`--verbose` flag can also be provided as part of the above command to output detailed description
+about how many pods can be scheduled on each node in the cluster. An example of verbose output on
+a 2-node cluster is as follows:
+
+```
+small-pod pod requirements:
+	- CPU: 150m
+	- Memory: 100Mi
+
+The cluster can schedule 52 instance(s) of the pod small-pod.
+
+Termination reason: Unschedulable: No nodes are available that match all of the following predicates:: Insufficient cpu (2).
+
+Pod distribution among nodes:
+small-pod
+	- 192.168.124.214: 26 instance(s)
+	- 192.168.124.120: 26 instance(s)
+
+```
+
+[admin-guide-cluster-capacity-pod]
+### Running cluster capacity in a pod
+
+Below is an example of a pod spec (`cluster-capacity-pod.yaml`) that runs cluster capacity image as a container. Note that
+input pod spec (`pod.yaml`) to cluster capacity analysis is mounted in a volume using `hostPath`.
+
+```yaml
+apiVersion: v1
+kind: Pod
+metadata:
+  name: cluster-capacity
+  labels:
+    name: cluster-capacity
+spec:
+  containers:
+  - name: cluster-capacity
+    image: openshift/origin-cluster-capacity
+    env:
+    - name: CC_INCLUSTER
+      value: "true"
+    command:
+    - "/bin/sh"
+    - "-ec"
+    - |
+      /bin/cluster-capacity --podspec=/test-pod/pod.yaml
+    volumeMounts:
+    - mountPath: /test-pod
+      name: test-volume
+  serviceAccountName: cluster-capacity-sa
+  volumes:
+  - name: test-volume
+    hostPath:
+      # directory location on host
+      path: /data
+```
+
+Also note that the above pod spec specifies the following environment variable which is required to
+let the cluster capacity command know that it is running inside a cluster as a pod.
+
+```yaml
+env:
+- name: CC_INCLUSTER
+  value: "true"
+```
+
+By default, this pod would be created in the `default` namespace by using `default` service account. However, the above
+pod spec refers to a different service account (`cluster-capacity-sa`) in the `default` namespace created as below:
+
+```
+$ oc create sa cluster-capacity-sa
+```
+
+This service account `cluster-capacity-sa` is given `cluster-admin` cluster scope role below so that the cluster capacity command
```

Comment:
cluster-admin is way more permission than is required.  Wouldn't cluster-reader work as well?

Ideally we should spec how to create a role with the actual permissions needed.